### PR TITLE
Add Contact-form section

### PR DESCRIPTION
**Pull request by naji-97**
### **The following requirments has benn made:**
- [ ]  Add a contact form to my portfolio with the help of my coding partner. 
- [ ] Add required field for username and email by using HTML 
- [ ] style the form with the CSS in accordance with figma Template

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,160 +2,217 @@
...
+                <h2>contact me</h2>
+
+                <p>
+                    if you have an application you are interested in developing, a
+                    features that you need built or a project that need coding. I'd love
+                    to help with it.
+                </p>
+            </div>
+            <div class="contact-me-form">
+                <form>
```
Comment (Yazino12) [inline index.html]: - [ ] In your current form condition, when I submit the form nothing happens 🤔. Please make sure that you use [Formspree Service](https://formspree.io/html) to collect data, and also to integrate your from with this service use the `action` and `method` attributes 📝.

image

File: index.html [hunk 2]
```diff
@@ -2,160 +2,217 @@
...
+                    <div class="email-input">
+                        <input type="email" name="email" class="input" placeholder="enter email"
+                            pattern="^[a-zA-Z0-9]+@domain\.[a-zA-z]+$" size="64" maxlength="64" required />
+                    </div>
+                    <br />
+                    <div class="text-area">
+                        <textarea name="message" type="text" class="input-textarea" cols="30" maxlength="500"
+                            placeholder="enter your message here" required></textarea>
+                    </div>
+                    <button type="button" value="Get In Touch" class="submit-button">
```
Comment (Yazino12) [inline index.html]: - [ ] In order to submit your form successfully ✔, make sure you set the type of your button to `submit` or leave it with no type attribute, as the default type is set to **submit** when no type attribute is specified 😊.

**Review by Yazino12 — CHANGES_REQUESTED** [review]
## Hi @naji-97 🖐🏾,

You've done well implementing some of the project requirements 🎉🎊🎉, but there are still some you need to work on to move to the next project 🔨.

### To highlight:
- Linters are passing ✔️
- Correct use of GitHub flow ✔️ 

# STATUS:  REQUIRED CHANGES ♻♻♻ 
TRY AGAIN
_Check the comments under the review 👇🏽👇🏽👇🏽._

## Optional suggestions

- [ ] Please make sure to add more descriptive points in to your pull request summary 👇🏽
image

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏 💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @Yazino12 when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -2,160 +2,217 @@
...
+                <h2>contact me</h2>
+
+                <p>
+                    if you have an application you are interested in developing, a
+                    features that you need built or a project that need coding. I'd love
+                    to help with it.
+                </p>
+            </div>
+            <div class="contact-me-form">
+                <form action="https://formspree.io/f/{form_id}" method="post">
```
Comment (emmiiorji) [inline index.html]: As requested by the previous reviewer, kindly create an account at with [formspree service](https://formspree.io/html) and use the id assigned to you to replace `form_id` in `https://formspree.io/f/{form_id}`.

File: index.html [hunk 4]
```diff
@@ -2,160 +2,217 @@
...
+                    <div class="username-input">
+                        <input type="text" name="username" class="input" placeholder="enter username" maxlength="30"
+                            required />
+                    </div>
+                    <br />
+                    <div class="email-input">
+                        <input type="email" name="email" class="input" placeholder="enter email"
+                            pattern="^[a-zA-Z0-9]+@domain\.[a-zA-z]+$" size="64" maxlength="64" required />
```
Comment (emmiiorji) [inline index.html]: Kindly use review the provided regular expression as it does not seem to match an actual email.

image

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @naji-97, 

# Status: Changes Required ♻️

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

A little bit more

 ## Good Highlights✔️

- Linter checks are passing
- Adhered to GitHub flow
- Good commit messages


 ## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by joskalenda — APPROVED** [review]
| STATUS: APPROVED ✅ |
|------------------------|
| Hi @naji-97 , 🥰|
| Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉  |
|  |

### Highlights
✅ ...Green linters
✅ ...

---------
### Optional suggestions

> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### Cheers and Happy coding!👏👏👏
-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**